homeassistant/components/zwave/__init__.py

```diff
@@ -103,7 +104,7 @@
 RENAME_NODE_SCHEMA = vol.Schema(
     {
         vol.Required(const.ATTR_NODE_ID): vol.Coerce(int),
```

Comment:
Those `const.ATTR_NODE_ID` could also be imported individually.

Reply:
Not from the core constants. It does not exist. There's more than just `ATTR_NODE_ID`. Are you suggesting to individually import then all?

Reply:
> Are you suggesting to individually import then all?

Exactly that. But to be fair, I have no issue if you consider this out of scope. Just something I noticed when reviewing the PR.

Reply:
I believe this to be out of scope. The point of me making these PRs is to have components use the core constants whenever possible, not to rework the coding style.